components/link/skin.css

```diff
@@ -30,8 +30,7 @@ governing permissions and limitations under the License.
   }
 }
 
-.spectrum-Link--quiet,
-/** @deprecated */.spectrum-Link--subtle {
+.spectrum-Link--secondary{
   color: inherit;
```

Comment:
That looks right. Not sure what to do on hover though, is it correct as-is? 
https://spectrum.adobe.com/page/link/#Secondary-variant

Reply:
hmm... This change was pulled in from RSP.

According to the description of Secondary variant on the spectrum-documentation site, the color should be [the same gray color as the paragraph text](https://spectrum.corp.adobe.com/page/link/#Secondary-variant).

Is it this reason that it used `inherit`? But I did find DNA tokens for  Secondary variant.

` --spectrum-link-secondary-text-color: var(--spectrum-alias-text-color);
   --spectrum-link-secondary-text-color-hover: var(--spectrum-alias-text-color-hover);
  --spectrum-link-secondary-text-color-key-focus: var(--spectrum-alias-text-color-hover);
`

Reply:
General rule of thumb we should try to abide by: Find DNA tokens and use them whenever available. For now, inheriting is displaying the correct colors, but if we change this in the future for *secondary links only* (no other type), then the change will not be picked up via DNA upgrade as expected.